### PR TITLE
Avoid re-using a variable for a different purpose

### PR DESCRIPTION
**Pull request by edg2s**
fraction starts off as an object then becomes a string

### CODE REVIEW

**Comment by edg2s** [comment]
I signed the CLA, need to retrigger?

File: src/currency/supplemental-override.js [hunk 1]
```diff
@@ -8,16 +8,13 @@ define([
...
 return function( currency, pattern, cldr ) {
-	var digits,
-		fraction = cldr.supplemental([ "currencyData/fractions", currency ]) ||
-			cldr.supplemental( "currencyData/fractions/DEFAULT" );
-
-	digits = +fraction._digits;
+	var fraction = "",
+		fractionData = cldr.supplemental([ "currencyData/fractions", currency ]) ||
+			cldr.supplemental( "currencyData/fractions/DEFAULT" ),
+		digits = +fractionData._digits;
```
Comment (rxaviers) [inline src/currency/supplemental-override.js]: `digits` depends on `fractionData` to be set. Therefore, I prefer it on a new line. Can you revert that part, please? (this is, `var digits,` then `digits = `)

**Comment by rxaviers** [comment]
Thanks for your contribution. PS: The CLA checker will only retrigger on new commits, but I can check it manually, don't worry.

File: src/currency/supplemental-override.js (hunk 1 above)
Comment (edg2s) [inline src/currency/supplemental-override.js]: sure

**Comment by rxaviers** [comment]
I'm cc'ing @jquery/globalize to review this code to avoid me being biased to think the former is cleaner given the fact I wrote it. :)

**Comment by jzaefferer** [comment]
Change looks good to me. Reusing a variable for different types seems bad indeed.

As for the CLA, it looks like there's multiple signatures with the same email address, which our automatic check [doesn't handle properly](https://github.com/jquery/jquery-license/issues/32). Changing the git author to the full name should fix the issue, since that's what the first signature had. Either way we've got a valid signature.

**Comment by rxaviers** [comment]
Thanks @edg2s for your contribution. Merged.

**Comment by rxaviers** [comment]
@jzaefferer thanks for your review.

**Pull request closed without merging**